js/main.js

```diff
@@ -31,4 +33,41 @@ Site = {
   },
 };
 
+Site.News = {
+  init: function() {
+    var _this = this;
+
+    _this.shuffleContainer = document.getElementById('shuffle-container');
+
+    _this.bind();
+  },
+
+  bind: function() {
+    var _this = this;
+
+    imagesLoaded(_this.shuffleContainer, function() {
+      _this.initShuffle();
+    });
+  },
+
+  initShuffle: function() {
+    var _this = this;
+
+    $('#shuffle-preloader').remove();
+    $(_this.shuffleContainer).css('opacity', 1)
```

Comment:
I wonder if this also works for the filter and ms props that autoprefixer generates for `opacity` on line `755` and `756` of `site.css`. perhaps its better to add remove a `hide` class than to change css props with js